config.toml

```diff
@@ -126,15 +126,34 @@ grpc_addr = 'http://127.0.0.1:9090'
 # listens on. Required
 websocket_addr = 'ws://127.0.0.1:26657/websocket'
 
+# The type of event source to use for getting events from the chain.
+# This setting can take one of two values:
+# 
+# - `websocket` for WebSocket
+# - `rpc` for RPC via the `/block_results` endpoint
+#
+# Default: 'websocket'
+event_source = 'websocket'
+
+# Delay until event batch is emitted if no NewBlock events have come yet.
+#
+# Note: Only applies when `event_source` is set to `websocket`.
+#
+# Default: 500ms
+batch_delay = '500ms'
+
+# The interval at which to poll for blocks when the `event_source` is `rpc`.
+# Note: Only applies when `event_source` is set to `rpc`.
+#
+# Default: 1 second
+poll_interval = '1s'
+
```

Comment:
as discussed, consider grouping the params, e.g.:
```
event_settings = {type = push, websocket_addr = 'ws://127.0.0.1:26557/websocket', interval = '1s'}
event_settings = {type = poll, interval = '1s'}
```

Reply:
I am now wondering if we should in fact explore two extra models:
- the current poll one you have in this PR (would be for users who don't have access to websocket server)
- a hybrid one where we only register for `NewBlock` event and act on it by fetching the `/block_results` (or `/tx_results` since we already get begin and end block events)

Reply:
Settings changes in https://github.com/informalsystems/hermes/pull/3323/commits/829348d7aa16231b159289f7f26d687ff3c763eb

Reply:
Yeah that sounds like a great idea!

Reply:
I don't see a `/tx_results` RPC endpoint, do you mean `/tx_search` + `/tx`?

Reply:
I have a [prototype](https://github.com/informalsystems/hermes/pull/3323) for a hybrid event source, but still using `/block_results` for now.